tests/covalent_dispatcher_tests/_core/data_manager_test.py

```diff
@@ -18,526 +18,526 @@
-    assert get_status_queue(dispatch_id) is q
-
-
-@pytest.mark.asyncio
-async def test_persist_result(mocker):
-    """
-    Test persisting the result object
-    """
-    result_object = get_mock_result()
-
-    mock_get_result = mocker.patch(
-        "covalent_dispatcher._core.data_manager.get_result_object", return_value=result_object
-    )
-    mock_update_parent = mocker.patch(
-        "covalent_dispatcher._core.data_manager._update_parent_electron"
-    )
-    mock_persist = mocker.patch("covalent_dispatcher._core.data_manager.update.persist")
-
-    await persist_result(result_object.dispatch_id)
-    mock_update_parent.assert_awaited_with(result_object)
-    mock_persist.assert_called_with(result_object)
-
-
-@pytest.mark.parametrize(
-    "sub_status,mapped_status",
-    [
-        (RESULT_STATUS.COMPLETED, RESULT_STATUS.COMPLETED),
-        (RESULT_STATUS.POSTPROCESSING_FAILED, RESULT_STATUS.FAILED),
-    ],
-)
-@pytest.mark.asyncio
-async def test_update_parent_electron(mocker, sub_status, mapped_status):
-    """
-    Test updating parent electron data
-    """
-    parent_result_obj = get_mock_result()
-    sub_result_obj = get_mock_result()
-    eid = 5
-    parent_dispatch_id = (parent_result_obj.dispatch_id,)
-    parent_node_id = 2
-    sub_result_obj._electron_id = eid
-    sub_result_obj._status = sub_status
-    sub_result_obj._result = 42
-
-    mock_node_result = {
-        "node_id": parent_node_id,
-        "end_time": sub_result_obj._end_time,
-        "status": mapped_status,
-        "output": sub_result_obj._result,
-        "error": sub_result_obj._error,
-    }
-
-    mocker.patch(
-        "covalent_dispatcher._core.data_manager.generate_node_result",
-        return_value=mock_node_result,
-    )
-
-    mock_update_node = mocker.patch("covalent_dispatcher._core.data_manager.update_node_result")
-    mocker.patch(
-        "covalent_dispatcher._core.data_manager.resolve_electron_id",
-        return_value=(parent_dispatch_id, parent_node_id),
-    )
-    mock_get_res = mocker.patch(
-        "covalent_dispatcher._core.data_manager.get_result_object", return_value=parent_result_obj
-    )
-    load_mock = mocker.patch("covalent_dispatcher._core.data_manager.load")
-    load_mock.sublattice_dispatch_id.return_value = "mock-sub-dispatch-id"
-    await _update_parent_electron(sub_result_obj)
-
-    mock_get_res.assert_called_with(parent_dispatch_id)
-    mock_update_node.assert_awaited_with(parent_result_obj, mock_node_result)
-
-
-def test_upsert_lattice_data(mocker):
-    """
-    Test updating lattice data in database
-    """
-    result_object = get_mock_result()
-    mocker.patch(
-        "covalent_dispatcher._core.data_manager.get_result_object", return_value=result_object
-    )
-    mock_upsert_lattice = mocker.patch("covalent_dispatcher._db.upsert.lattice_data")
-    upsert_lattice_data(result_object.dispatch_id)
-    mock_upsert_lattice.assert_called_with(result_object)
+# """
```

Comment:
Not sure why these tests are commented out - Remove if these tests no longer make sense

Reply:
The existing `{data_manager,dispatcher,runner}_test.py` are renamed temporarily to `tmp_{data_manager,dispatcher,runner}_test.py` and will be largely rewritten in the next [patch](https://github.com/AgnostiqHQ/covalent/pull/1729).